docs/UserGuide.md

```diff
@@ -224,6 +233,20 @@ Format: `list`
 
 <br>
 
+### Listing all details of a patient: `focus -p`
+
+Shows all details of a specified patient.
+
+Format: `focus -p PATIENT_INDEX`
+* The index refers to the index number shown in the displayed patient list.
+* The index **must be a positive integer** 1, 2, 3, …
```

Comment:
I think these 2 lines can be omitted since they are already mentioned in command format above. No need to fix if pressed for time since I will do a final check to standardise this.